lib/Sass/exports/_variables-export.scss

```diff
@@ -43,7 +43,7 @@
   colorPrimary: $color-primary;
   colorSecondary: $color-secondary;
   textLight: $text-light; // essentially #fff until we sort out the colour palette from vic
-  textLightTranslucent: rgba($text-light, 0.5);
+  textLightTranslucent: rgba($text-light, 0.3);
```

Comment:
This is only used for the mobile hamburger menu border. So I think it is safe to change it.